src/RuleLocationCollection.php

```diff
@@ -39,7 +39,9 @@ class RuleLocationCollection extends RuleCollection
 {
     public $stop_on_first_match = true;
     public static $rightname    = 'rule_location';
-
+    public $orderby             = 'name';
```

Comment:
I do not know why it is still possible to change the field used to define rules display order, but it completely breaks the ability to sort them. Indeed, while it is still possible to use the drag&drop feature (see image), their ranking is changed but they are always loaded and evaluated based on a name sorting (and reloading the page always show them ordered by name)
![image](https://user-images.githubusercontent.com/33253653/216353672-3559aba6-242b-43e1-8a62-c880398486c2.png)

```suggestion
```

Reply:
Cédric, you're right : I missed the drag&drop feature, so forget the adding of **$orderby** suggestion
FYI, I took the **$orderby** instruction from the RuleRightCollection class (need to be corrected as well ?)

I still suggest the adding of **$menu_option** to have the breadcrumb trail updated (see image)
![image](https://user-images.githubusercontent.com/120382993/216925549-695f0049-f9d3-4197-8e89-6d7183c6d64f.png)

Reply:
> FYI, I took the **$orderby** instruction from the RuleRightCollection class (need to be corrected as well ?)

#13988 will remove ability to override default order field.